server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -91,9 +94,13 @@ pub struct RequisitionRow {
     #[diesel(column_name = type_)]
     pub r#type: RequisitionType,
     pub status: RequisitionStatus,
+    #[ts(as = "String")]
     pub created_datetime: NaiveDateTime,
```

Comment:
NaiveDateTime does not implement TS (should check maybe there is a features flag that implements it